interpreter/cling/include/cling/Interpreter/InterpreterCallbacks.h

```diff
@@ -125,6 +125,13 @@ namespace cling {
     virtual bool LookupObject(const clang::DeclContext*, clang::DeclarationName);
     virtual bool LookupObject(clang::TagDecl*);
 
+    /// \brief This callback is invoked whenever the interpreter failed to load a library.
+    ///
+    /// \param[in] - Error message and parameters passed to loadLibrary
+    /// \returns true if the error was handled.
+    virtual bool LibraryLoadingFailed(std::string errmessage, std::string libStem,
```

Comment:
Can you comment out the parameters name to avoid some compiler complaining about unused parameters?  Thanks.